components/cart/open-cart.tsx

```diff
@@ -1,24 +1,18 @@
-import { ShoppingCartIcon, XMarkIcon } from '@heroicons/react/24/outline';
+import { ShoppingCartIcon } from '@heroicons/react/24/outline';
```

Comment:
Can you rename the exported component to `OpenCart` please?

Sorry I can't comment on the actual line as it didn't change. 🫤